gulp/tasks/build.coffee

```diff
@@ -18,15 +18,16 @@ gulp.task 'rebuild',
 gulp.task 'deploy', 'Deploy project to Google App Engine.', ['build'], ->
   options = minimist process.argv
```

Comment:
there are some issues with the [minimist](https://github.com/substack/minimist/issues/54) and in our case when the `--no-promote` is being used.. it is translated to `promote: false` :/